### PR TITLE
fix: shift the buttons to the bottom in LNURLpay screen view

### LINKED ISSUE #2663: [BUG] Confirm and cancel buttons should be on bottom
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

[BUG] Confirm and cancel buttons should be on bottom.

photo_5098406371589729167_x

### Screenshots [optional]

_No response_

### Steps To Reproduce

1- Go to https://www.stacksats.how/podcast and click on Alby extension button

### Expected behavior

Confirm and cancel buttons on bottom

### Alby information

Alby: 3.0.0

### Device information

_No response_

### Additional context

_No response_

### Are you working on this?

No

### PR DESCRIPTION
**Pull request by Rithvik-padma**
### Describe the changes you have made in this PR

Shifted the confirm and cancel buttons in the LNURL screen view to the bottom

### Link this PR to an issue

Fixes #2663 

### Type of change

- `fix`: Bug fix (non-breaking change which fixes an issue)

### Screenshots of the changes 

Before the changes 

image

After the changes

image

### How has this been tested?

Tested the screens manually by trying to pay on some sites in LNURL mode.

### Checklist

- [x] Self-review of changed code
- [x] Manual testing
- [x] Responsive layout

### CODE REVIEW

**Comment by Rithvik-padma** [comment]
@stackingsaunter @rolznz Can u review this PR.
Thanks

File: src/app/screens/LNURLPay/index.tsx [hunk 1]
```diff
@@ -517,7 +520,7 @@ function LNURLPay() {
                         )}
                       </div>
                     )}
-                    <div className="mt-2 dark:border-white/10">
+                    <div className="absolute bottom-0 w-full dark:border-white/10">
```
Comment (reneaaron) [inline src/app/screens/LNURLPay/index.tsx:523]: Please don't use absolute positioning, as that will lead to other problems. (see screenshot) 

Have a look at other screens where the buttons are already aligned at the bottom (e.g. in some prompt screens)

image

**Review by reneaaron — CHANGES_REQUESTED** [review]
Thanks for the PR, just did a first review! :100:

File: src/app/screens/LNURLPay/index.tsx (hunk 1 above)
Comment (Rithvik-padma) [reply src/app/screens/LNURLPay/index.tsx:523]: got it, will make the changes for it!

Comment (Rithvik-padma) [reply src/app/screens/LNURLPay/index.tsx]: @reneaaron Made the changes can u review it now.
Thanks!

**Comment by Rithvik-padma** [comment]
@reneaaron Made the changes, can u review it now.
Thanks!

**Review by rolznz — APPROVED** [review]
tACK

**Comment by Rithvik-padma** [comment]
> tACK

@reneaaron Can u review these changes too.
Thanks!

**Pull request merged**